content/en/docs/contributing/localization.md

```diff
@@ -18,22 +18,44 @@ guidance offered in this section.
 
 ### Summary / TL;DR {#summary}
 
-Do translate:
+#### ✅ Do {#do}
 
-- [Front matter][] fields `title`, `linkTitle`, and `description`
-- Page content, including the text fields in Mermaid [diagrams](#images)
+<div class="border-start border-success bg-success-subtle">
 
-Do **NOT**:
+- **Translate**:
+  - Page content, including:
+    - Mermaid [diagram](#images) text fields
+  - [Front matter][] fields `title`, `linkTitle`, and `description`
```

Comment:
```suggestion
  - [Front matter][] values for the fields `title`, `linkTitle`, and `description`
```

Reply:
Actually, it is both field _keys_ and _values (yes, I've had some folks translate the field key names too). So I'd leave this as is. WDYT?

Reply:
Translating keys won't break them?

Reply:
Yes it does, but that didn't stop folks from translating them :).

Reply:
Oops, I read this out of context. I'm adding your suggestion (which supports the idea of not translating key names)!